sdk/storage/storage-file-share/CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Release History
 
+## 12.27.0-beta.1 (2025-02-13)
+
+### Features Added
+
+- Added support for service version 2025-05-05.
+- Added support for NFS over REST.
+
+### Breaking Changes
+
+- The following APIs no longer send the x-ms-file-permission-key, x-ms-file-attributes, x-ms-file-creation-time, and x-ms-file-last-write-time request headers by default. These headers have been optional in the REST API since x-ms-version 2021-06-08:
```

Comment:
Should we mention, the behavior of the server is the same.

Reply:
If there are truly breaking changes shouldn't you release this as version 13?

Reply:
Overall behavior doesn't change. Customer should not need to worry about it. Removed the breaking change description.